package.json

```diff
@@ -65,6 +70,7 @@
     "broccoli-merge-trees": "^1.1.0",
     "broccoli-string-replace": "^0.1.1",
     "chalk": "^1.1.0",
+    "electron-forge": "^2.8.0",
     "electron-packager": "^8.3.0",
```

Comment:
This is now an implicit dependency, unless for some odd reason we need this as a direct dependency.